xskillscore/core/contingency.py

```diff
@@ -70,31 +70,26 @@ class Contingency:
 
     Examples
     --------
-    >>> a = xr.DataArray(
-    ...     np.random.normal(size=(3, 3)),
-            coords=[('x', np.arange(3)), ('y', np.arange(3))]
-    ... ).to_dataset(name='test1')
-    >>> b = xr.DataArray(
-    ...    np.random.normal(size=(3, 3)),
-    ...    coords=[('x', np.arange(3)), ('y', np.arange(3))]
-    ... ).to_dataset(name='test1')
-    >>> a['test2'] = xr.DataArray(np.random.normal(size=(3,3)),
-    ...                           coords=[('x', np.arange(3)), ('y', np.arange(3))])
-    >>> b['test2'] = xr.DataArray(np.random.normal(size=(3,3)),
-    ...                           coords=[('x', np.arange(3)), ('y', np.arange(3))])
-    >>> category_edges_a = np.linspace(-2,2,5)
-    >>> category_edges_b = np.linspace(-3,3,5)
-    >>> Contingency(a, b, category_edges_a, category_edges_b, dim=['x','y'])
+    >>> da = xr.DataArray(np.random.normal(size=(3, 3)),
+    ...                   coords=[("x", np.arange(3)), ("y", np.arange(3))])
+    >>> o = xr.Dataset({"var1": da, "var2": da})
+    >>> f = o * 1.1
+    >>> o_category_edges = np.linspace(-2, 2, 5)
+    >>> f_category_edges = np.linspace(-3, 3, 5)
+    >>> xs.Contingency(o, f,
+    ...                o_category_edges, f_category_edges,
+    ...                dim=['x', 'y']) # doctest: +SKIP
```

Comment:
Why skip here?

Reply:
See https://github.com/xarray-contrib/xskillscore/pull/302#issuecomment-832745660

Reply:
Couldn't work out elipsis to handle the randomness of var1 and var2 ordering.

The output is printed below anyway so you know what to expected